pycbc/distributions/sky_location.py

```diff
@@ -243,6 +345,26 @@ def normalize_azimuth(self, phi):
         phi[phi >= 2 * numpy.pi] -= 2 * numpy.pi
         return phi
 
+    def to_uniform_patch(self, coverage):
+        """Apply the coverage criterion on the skymap
+        """
+        uniform_patch = self
```

Comment:
If you do this, you will modify the original distribution, which is not ideal. You probably need to do a "deep copy" here: https://docs.python.org/3/library/copy.html